seed/php-sdk/alias-extends/src/SeedClient.php

```diff
@@ -29,9 +29,9 @@ public function __construct(
         ?array $options = null,
     ) {
         $defaultHeaders = [
-            "X-Fern-Language" => "PHP",
-            "X-Fern-SDK-Name" => "Seed",
-            "X-Fern-SDK-Version" => "0.0.1",
+            'X-Fern-Language' => 'PHP',
```

Comment:
To confirm, we'll use single quotes globally for strings?

Reply:
Yes, that's the plan! I added a note to the summary - we'll only want to use double quotes when we need string interpolation, e.g. when the API key is specified in-line like so:

```php
$defaultHeaders = [
  'Authorization' => "Bearer $apiKey"
]
```